### PR TITLE
Fix/upgw 566 fix(validators): add valueMoreThanIn & valueLessThanIn validators

### PR DESCRIPTION
**Pull request by oscaryald**

### CODE REVIEW

File: packages/components/validator-processing/validator-processing.service.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const XM_VALIDATOR_PROCESSING_CONTROL_ERRORS_TRANSLATES: XmControlErrorsT
     minDate: marker('xm-validator-processing.validators.minDate'),
     languageRequired: marker('xm-validator-processing.validators.languageRequired'),
     minArrayLength: marker('xm-validator-processing.validators.minArrayLength'),
+    valueLessThanIn: marker('xm-control-errors.validators.pattern'),
```
Comment (EKrasovskiy) [inline packages/components/validator-processing/validator-processing.service.ts:18]: please add translation for "valueMoreThanIn"

Comment (oscaryald) [reply packages/components/validator-processing/validator-processing.service.ts:18]: your right, thanks. done

**Review by EKrasovskiy — APPROVED** [review]

File: packages/components/validator-processing/validator-processing.service.ts (hunk 1 above)
Comment (Kaskyi) [reply packages/components/validator-processing/validator-processing.service.ts:18]: src/i18n/[en,ru,uk]/core.json.xm-validator-processing": { "validators

Comment (oscaryald) [reply packages/components/validator-processing/validator-processing.service.ts:18]: @Kaskyi , updated

**Review by Kaskyi — APPROVED** [review]

**Pull request merged**